### PR TITLE
Add alias_sensor to user visibile/editable keys.

### PR DESCRIPTION
**Pull request by jasebell**

### CODE REVIEW

File: src/clj/kixi/hecuba/data/sensors.clj [hunk 1]
```diff
@@ -158,21 +166,34 @@
...
 (defn encode
   ([sensor]
-     (encode sensor false))
+   (encode sensor false))
   ([sensor remove-pk?]
-     (-> sensor
-         (cond-> (:user_metadata sensor) (user-metadata (:synthetic sensor)))
-         (cond-> remove-pk? (dissoc :device_id :sensor_id)))))
+   (-> sensor
+       (cond-> (:user_metadata sensor) (user-metadata (:synthetic sensor)))
```
Comment (elisehuard) [inline src/clj/kixi/hecuba/data/sensors.clj]: I didn't know cond-> until today, however looking at the docs you could probably put all that in one cond-> ?

Comment (jasebell) [inline src/clj/kixi/hecuba/data/sensors.clj]: I'm not sure why there was two cond-> to start off with, I'd rather leave them in their original state know that the one we've added works and doesn't affect any other conditions.

**Comment by elisehuard** [comment]
apart from nitpick :shipit:

File: src/clj/kixi/hecuba/data/sensors.clj (hunk 1 above)
Comment (elisehuard) [inline src/clj/kixi/hecuba/data/sensors.clj]: fair enough

**Pull request merged**